src/ROUTES.js

```diff
@@ -25,6 +25,9 @@ export default {
     getIouRequestRoute: reportID => `iou/request/${reportID}`,
     IOU_BILL: 'iou/split/:reportID',
     getIouSplitRoute: reportID => `iou/split/${reportID}`,
+    IOU_DETAILS: 'iou/details',
+    IOU_DETAILS_WITH_IOU_REPORT_ID: 'iou/details/:chatReportID/:iouReportID/',
```

Comment:
While `iou/:chatReportID/:iouReportID/` would be a bit cleaner, it's a bit confusing as `iou/split/` and `iou/request` also use the `iou/` route.

Reply:
No, currently you cannot view an IOU outside of the currently viewed report.

Reply:
Ah gotcha. It's possible, but switching to this route simplified the logic IMO, as having both IDs available immediately after opening the Modal lets us retrieve data straight away, rather than awaiting the first Onyx response. I discussed the choices a bit [here](https://expensify.slack.com/archives/C01GTK53T8Q/p1620236167421500).